tests/utilities/test_all_gather_grad.py

```diff
@@ -58,13 +58,17 @@ def training_epoch_end(self, outputs) -> None:
             self.training_epoch_end_called = True
             losses = torch.stack([x["loss"] for x in outputs])
             gathered_loss = self.all_gather({
+                "losses_tensor_int": torch.tensor([1, 2, 3]),
+                "losses_tensor_float": torch.tensor([1., 2., 3.]),
```

Comment:
but these tensors are contiguous already. You need a multi dimensional tensor, and you can easily make it non-contiguous by adding a transpose, like this:

`torch.rand(2,2).t()`

Reply:
Thanks !